src/index.js

```diff
@@ -2,9 +2,32 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import About from './Components/About/About'
+import Privacy from './Components/Privacy/Privacy'
+import Legal from './Components/Legal/Legal'
+
 import * as serviceWorker from './serviceWorker';
+import { BrowserRouter as Router, Route, Redirect, Switch }
+from 'react-router-dom';
+import { createStore} from 'redux';
+import history from "./history";
+import { Provider } from 'react-redux';
+
+
+
+ReactDOM.render(
+   <Router history={history}>
+     <Switch>
+       <Route path="/" exact component={App} />
+       <Route path="/home" exact component={App} />
+       <Route path="/about" exact component={About} />
+       <Route path="/privacy" exact component={Privacy} />
+       <Route path="/legal" exact component={Legal} />
+
+     </Switch>
+   </Router>,
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ document.getElementById('root'));
 
 // If you want your app to work offline and load faster, you can change
```

Comment:
REMOVE BOILERPLATE COMMENTS OR CREATE REACT APP FROM SCRATCH OR TRY TO MASK ME AND SHOW  THAT YOU DIDN'T USED STRUCTURE FROM SOMEONE !